index.js

```diff
@@ -7,12 +7,134 @@ function ask(questionText) {
   });
 }
 
+function randomInt(max, min) {
+  return Math.floor(min + (Math.random() * (max - min + 1)))
+}
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+gameMode = await ask('Let\'s play a game of Guess the Number.\nFirst, tell me who should do the guessing.\nEnter "me" if you (the human) want to guess,\nEnter "you" if you want me (the computer) to guess.\n Enter "me" or "you": ')
+
+if (gameMode === "you"){
+    console.log("\nOk, I'll guess a number that you think of.\nBefore you choose a number, tell me the range I should guess in.")
+
+    let rangeMin = NaN
```

Comment:
I would recommend starting these as `0` rather than `NaN`. You've got the right idea starting them as a falsey value of the correct type, but `NaN` tends to creep through your code and cause unexpected bugs, and `0` is also falsey.